src/vs/workbench/browser/parts/activitybar/activitybarPart.ts

```diff
@@ -206,6 +200,16 @@ export class ActivitybarPart extends Part {
 		container.style.borderLeftColor = !isPositionLeft ? borderColor : null;
 	}
 
+	private getActivitybarItemColors(theme: ITheme): ICompositeBarColors {
+		return <ICompositeBarColors>{
+			backgroundColor: theme.defines(ACTIVITY_BAR_INACTIVE_FOREGROUND) ? theme.getColor(ACTIVITY_BAR_INACTIVE_FOREGROUND) : theme.getColor(ACTIVITY_BAR_FOREGROUND).transparent(0.6),
```

Comment:
@sandy081 this condition should not be needed once the inactive color derives from the active one

Reply:
@bpasero This condition was added to not to break existing themes